sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/DistributeConstants.java

```diff
@@ -106,6 +106,16 @@
      */
     static final String PARAMETER_UPDATE_SETUP_FAILED = "update_setup_failed";
 
+    /**
+     * API parameter for install identifier.
+     */
+    static final String PARAMETER_INSTALL_ID = "install_id";
+
+    /**
+     * API parameter for last installed release hash.
+     */
+    static final String PARAMETER_LAST_INSTALLED_RELEASE_HASH = "last_installed_release_hash";
```

Comment:
Aren't we supposed to send release identifier instead? There is a one to many relationship between hash and identifiers: multiple release ids can have the exact same hash as its only based on version.